specification/metadata/datageneratorversion.md

```diff
@@ -0,0 +1,17 @@
+# Data Generator Version
+
+A version specific to the data generator for providers which support multiple FOCUS datasets with different provider columns, or for providers who release multiple versions of their generated datasets between FOCUS Versions.
+
+The DataGeneratorVersion SHOULD be provided in the metadata, when multiple datasets using the same FOCUS version are provided with different provider columns. DataGeneratorVersion MUST be of type String and MUST NOT contain null values.
```

Comment:
I would vote for a "MUST be provided", because it's needed for practitioners to identify different versions, for example in case of bug fix.

Reply:
```suggestion
The DataGeneratorVersion MUST be provided in the metadata, when multiple datasets using the same FOCUS version are provided with different provider columns. DataGeneratorVersion MUST be of type String and MUST NOT contain null values.
```